oblib/data_model.py

```diff
@@ -1295,6 +1301,8 @@ def set(self, concept_name, value, **kwargs):
             number of places past the decimal point to be considered precise. (For
             decimal values only. Only one of precision or decimals is accepted, not
             both. Defaults to decimals=2.)
+          fact_id: string
```

Comment:
why `fact_id` here instead of `id`?  If `fact_id` is consistent with other uses of the kwarg `id` I think it's helpful to keep the same kwarg.

Reply:
In the Fact class the id is simply id (Fact.id).  But in the OBInstance class it needs to be spelled out as OBInstance.fact_id because there may be other types of id's.

Reply:
We anticipate context_id or ?

Reply:
Not sure but theoretically context_id or unit_id could be added to the OBInstance.set() method.  Right now the context is created and passed in as an object and unit_name is used instead of unit_id.  Jono might have more insight into whether the id's are potential future parameters.

Reply:
makes sense